6-CODE-SPLITTING/webpack.config.js

```diff
@@ -1,13 +1,27 @@
+const BundleAnalyzerPlugin =
+  require("webpack-bundle-analyzer").BundleAnalyzerPlugin;
+
 module.exports = {
-  mode: 'production',
+  mode: "production",
   entry: {
-    script1: './src/script1.js',
-    script2: './src/script2.js',
+    script1: "./src/script1.js",
+    script2: "./src/script2.js",
   },
   output: {
-    filename: '[name].bundle.js',
+    filename: "[name].bundle.js",
   },
   optimization: {
-    minimize: false,
-  }
+    minimize: true,
+    splitChunks: {
```

Comment:
Done. Split Chunks Plugin set up correctly, dist contains vendors.bundle.js with Lodash inside. script1.bundle.js and script2.bundle.js doesn't contain Lodash